run_sets/testsuite.yml

```diff
@@ -18,12 +17,29 @@
 - features/centos_tradclient.feature
 - features/centos_salt.feature
 - features/centos_salt_ssh.feature
+
 ## CORE feature end ###
 
 # SECONDARY FEATURE:
 
-# this feature can be runned in random order.
-# FIXME: MAKE THIS HAPPEN
+# PREFIX:
+
+# type (mandatory)
```

Comment:
If we're thinking about renaming the features files, then i would suggest to rename also the core features, something like `features/core_push_package.feature`.